tutorials/benchmark_simulation.ipynb

```diff
@@ -236,7 +236,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model = FlowBenchmark2dCase3aModel(
</pre><p>I think we should keep some identifier that this is the permeability model, to distinguish from the other model above. To clarify: permeability_model or model_0, model_1 is fine, while perm_setup, perm_m etc. are not allowed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I revert the respective two files, which previously used the correct formatting with prefix and suffix.